internal/config/config.go

```diff
@@ -23,8 +23,10 @@ var (
 	K9sConfigFile = filepath.Join(K9sHome(), "config.yml")
 	// K9sLogs represents K9s log.
 	K9sLogs = filepath.Join(os.TempDir(), fmt.Sprintf("k9s-%s.log", MustK9sUser()))
+	// k9sDumpDirPath represents a base directory where K9s screen dumps will be persisted.
+	k9sDumpDirPath = GetDumpConfigDirPath(K9sConfigFile)
 	// K9sDumpDir represents a directory where K9s screen dumps will be persisted.
-	K9sDumpDir = filepath.Join(os.TempDir(), fmt.Sprintf("k9s-screens-%s", MustK9sUser()))
+	K9sDumpDir = filepath.Join(k9sDumpDirPath, fmt.Sprintf("k9s-screens-%s", MustK9sUser()))
```

Comment:
Think this should be specified as K9sDefaultDumpDir and provide an config accessor to return the correct dump location while performing an actual dump operation as it can be overriden either via cli args or configuration.

Reply:
I made the required changes.